data/flyway/run.sh

```diff
@@ -0,0 +1,2 @@
+JDBC_URL=`echo $VCAP_SERVICES | jq -r '.["user-provided"][0]["credentials"]["JDBC_URL"]'`
+/home/vcap/app/flyway/bin/flyway -n -locations=filesystem:/home/vcap/app/flyway/migrations -url="${JDBC_URL}" info && echo SUCCESS
```

Comment:
Need to change "info" to "migrate" during the real run.

Reply:
Nice catch, @rjayasekera!